packages/clay-multi-step-nav/README.mdx

```diff
@@ -14,43 +14,54 @@ import {
 <div class="nav-toc-absolute">
 <div class="nav-toc">
 
--   [ClayMultiStepNav](#claymultistepnav)
--   [ClayMultiStepNavWithBasicItems](#claymultistepnavwithbasicitems)
+-   [Composing](#composing)
+-   [Collapsable Steps](#collapsable-steps)
 -   [API](#api)
 
 </div>
 </div>
 
-### ClayMultiStepNav
+# Composing
+
+Multi Step Nav is a progress bar used to divide the main task into subtasks to help users complete long processes.
+Multi Step Nav is used when a major or big task has to be divided into smaller task, with the aim of letting the user breath in the process and providing them with a sense of progression.
```

Comment:
I wouldn't classify this under the "#composing" header and this info also feels sort of redundant from the "description" in the frontmatter

Reply:
Same as [comment above](https://github.com/liferay/clay/pull/3226#discussion_r425718518). Not rendered in React Component page.